--- a/modules/Dee/moc_plugin.cpp	2014-09-09 16:04:59.443249046 -0400
+++ b/modules/Dee/moc_plugin.cpp	2014-09-09 16:01:16.901965193 -0400
@@ -75,12 +75,12 @@
     if (!_clname) return 0;
     if (!strcmp(_clname, qt_meta_stringdata_DeePlugin.stringdata))
         return static_cast<void*>(const_cast< DeePlugin*>(this));
-    return QDeclarativeExtensionPlugin::qt_metacast(_clname);
+    return qt_metacast(_clname);
 }
 
 int DeePlugin::qt_metacall(QMetaObject::Call _c, int _id, void **_a)
 {
-    _id = QDeclarativeExtensionPlugin::qt_metacall(_c, _id, _a);
+    _id = qt_metacall(_c, _id, _a);
     if (_id < 0)
         return _id;
     return _id;
